docs/src/examples/components/List/Content/ListExampleMedia.shorthand.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { List } from '@stardust-ui/react'
+
+const imgStyle = { display: 'block', width: '2rem', borderRadius: '999rem' }
+
+const items = [
+  {
+    key: 'irving',
+    media: <img src="public/images/avatar/small/matt.jpg" style={imgStyle} />,
```

Comment:
Now that we have an `<Image avatar />` component, we should use that here instead of the `img` tag.  The goal is to have no inline styles in doc examples and using the `Image` would allow us to do that.

Just import `Image` and replace the `img` components with our `Image`.  Then, remove the inline styles and apply the `avatar` prop.  Example:

https://stardust-ui.github.io/react/components/image#types-avatar

Let's do this for all examples using `img` like this.